lib/rules/selector-type-no-unknown/__tests__/index.js

```diff
@@ -77,6 +77,18 @@ testRule(rule, {
   }, {
     code: "abs {}",
     description: "MathML tags",
+  }, {
+    code: ".foo /deep/ .bar {}",
+    description: "shadow-piercing descendant combinator",
+  }, {
+    code: ".foo /dEeP/ .bar {}",
+    description: "shadow-piercing descendant combinator",
+  }, {
+    code: ".foo /DEEP/ .bar {}",
+    description: "shadow-piercing descendant combinator",
+  }, {
+    code: ".foo /for/ .bar {}",
+    description: "reference combinators",
```

Comment:
@davidtheclark but seem it is deprecated selector, i don't see infomation about this in other spec, btw some project use `/deep/` and be good just ignore this